app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -205,6 +206,13 @@ class SessionControlView(
                     super.onLayoutCompleted(state)
 
                     JumpBackInCFRDialog(view).showIfNeeded()
```

Comment:
Do we need to worry that jump back in CFR be shown at the same time?  Maybe we need to extract the `shouldShowJumpBackInCFR` out of the dialog code and have it checked here so we can show one but not both?

Reply:
I was wondering the same. But this is exactly what needs to be done in https://mozilla-hub.atlassian.net/browse/FNXV2-20988

Reply:
This is something I am also running into where a CFR could appear when the MR onboarding dialog for upgraded users is shown. I think we want to look at the order of when CFR and onboarding should appear and how and when they should be shown between each other. I don't think we need to intervene in this PR. I would suggest that we just land this behind a feature flag and file a followup.

Reply:
My idea with the CFRPresenter/Manager was to have a central point for controlling all CFRs for a particular screen / bigger feature - the presenter would be responsible for handling more CFRs, not just one.
In the case of more CFRs shown on the homescreen having one CFRPresenter/Manager that would control them all - when to appear, in what order, etc could help. But yeah, seems like a bigger refactoring.